src/main/java/uk/gov/justice/hmpps/prison/service/ImageService.java

```diff
@@ -47,5 +51,41 @@ public ImageDetail findImageDetail(final Long imageId) {
             .map(i -> fullSizeImage ? i.getFullSizeImage() : i.getThumbnailImage());
     }
 
+    @PreAuthorize("hasRole('SYSTEM_USER')")
+    @HasWriteScope
+    @Transactional
+    public ImageDetail putImageForOffender(final String offenderNumber, final boolean fullSizeImage,  final String imageData) {
```

Comment:
This should have `@VerifyOffenderAccess(overrideRole = 'ROLE_IMAGE_UPLOAD')`

Reply:
Added